appengine-java17-bundled-services/datastore/src/main/java/com/example/appengine/AbstractGuestbook.java

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Are these copied instead of symlinked? If so, please update to eliminate duplication.